addons/sourcemod/scripting/include/superzombiefortress.inc

```diff
@@ -3,7 +3,7 @@
 #endif
 #define _superzombiefortress_included
 
-enum eWeaponsRarity
+enum WeaponRarity
```

Comment:
May well then also rename `eWeaponsRarity_Common` to `WeaponsRarity_Common` etc while were breaking sub-plugins with `eWeaponsRarity`